### PR TITLE
[EXA] Add spec for exa

### PR DESCRIPTION
**Pull request by alexnaiman**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Added new spec for exa 
**What is the current behavior? (You can also link to an open issue here)**
There isn't any spec for exa
**What is the new behavior (if this is a feature change)?**
This PR adds the exa
**Additional info:**

### CODE REVIEW

File: dev/exa.ts [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+      description: "show hidden and 'dot' files",
+      name: ["-a", "--all"],
+    },
+    {
+      description: "list directories like regular files",
+      name: ["-d", "--list-dirs"],
+    },
+    {
+      description: "limit the depth of recursion",
+      name: ["-L", "--level DEPTH"],
```
Comment (cstrnt) [inline dev/exa.ts:82]: I think DEPTH should not be included in the name because it is in fact an argument, right?

File: dev/exa.ts [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+      description: "which field to sort by",
+      name: ["-s", "--sort"],
+      args: {
+        name: "sort field",
+        suggestions: [
+          {
+            name: "name",
+          },
+          {
+            name: "Name",
```
Comment (cstrnt) [inline dev/exa.ts:98]: Does is make a difference for exa?

File: dev/exa.ts [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+      description: "list directories before other files",
+      name: ["--group-directories-first"],
+    },
+    {
+      description: "list only directories",
+      name: ["-D", "--only-dirs"],
+    },
+    {
+      description: "glob patterns (pipe-separated) of files to ignore",
+      name: ["-I", "--ignore-glob"],
```
Comment (cstrnt) [inline dev/exa.ts:152]: I think here is an `arg` required

File: dev/exa.ts [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+      name: ["-m", "--modified"],
+    },
+    {
+      description: "show number of file system blocks",
+      name: ["-S", "--blocks"],
+    },
+    {
+      description:
+        "which timestamp field to list (modified, accessed, created)",
+      name: ["-t", "--time FIELD"],
```
Comment (cstrnt) [inline dev/exa.ts:193]: Missing arg

**Review by cstrnt — CHANGES_REQUESTED** [review]

File: dev/exa.ts (hunk 1 above)
Comment (alexnaiman) [reply dev/exa.ts:82]: My bad, I generated the spec using a script and some things I had to manually correct

File: dev/exa.ts (hunk 2 above)
Comment (alexnaiman) [reply dev/exa.ts:98]: Yes, it actually does 😄 

>  Fields starting with a capital letter will sort uppercase before  lowercase:  'A'  then 'B' then 'a' then 'b'.  Fields starting with a lowercase letter will mix them: 'A' then 'a' then 'B' then 'b'.

File: dev/exa.ts (hunk 3 above)
Comment (alexnaiman) [reply dev/exa.ts:152]: Added right below

File: dev/exa.ts (hunk 1 above)
Comment (cstrnt) [reply dev/exa.ts]: No worries 😄

**Review by cstrnt — APPROVED** [review]
Awesome!

**Review by QuiiBz — APPROVED** [review]

**Review by mattschrage — APPROVED** [review]
This is awesome! Thanks for contributing @alexnaiman 🎉

**Pull request merged**